### PR TITLE
New Wizard layout

### PR DESCRIPTION
**Pull request by blankdots**
### Description

New layout for wizard.

image

this issue does not include accordion layout for selecting objects

### Type of change

- [x] New feature (non-breaking change which adds functionality)

### Changes Made

1. modified stepper
2. modified placement of back, next controls
3. add the persistent footer for wizard control/

### Testing

- [x] Tests do not apply

### CODE REVIEW

File: src/components/NewDraftWizard/WizardComponents/WizardFooter.js [hunk 1]
```diff
@@ -5,18 +5,33 @@ import { useDispatch, useSelector } from "react-redux"
 import Link from "@material-ui/core/Link"
 import { Link as RouterLink } from "react-router-dom"
 import Button from "@material-ui/core/Button"
-import { decrement, increment, reset } from "../../../features/wizardStepSlice"
+import { reset } from "../../../features/wizardStepSlice"
```
Comment (otahontas) [inline src/components/NewDraftWizard/WizardComponents/WizardFooter.js:8]: you can also use absolute imports: `mport { reset } from "features/wizardStepSlice"`

File: src/components/NewDraftWizard/WizardComponents/WizardFooter.js [hunk 2]
```diff
@@ -40,44 +55,43 @@ const WizardFooter = ({ nextButtonRef }: nextButtonRefProp) => {
...
+        </div>
+        {wizardStep >= 0 && (
+          <div>
+            <Button
+              variant="contained"
+              color="primary"
+              disabled={wizardStep >= 1 ? false : true}
+              className={classes.footerButton}
+              onClick={async () => {
+                await nextButtonRef.current.submitForm()
```
Comment (otahontas) [inline src/components/NewDraftWizard/WizardComponents/WizardFooter.js:81]: Note for later: this should probably be something else than form submission? maybe post folder content to backend,

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
+html,
```
Comment (otahontas) [inline src/style.css:1]: Maybe combine this with createMuiTheme in index.js?

Comment (blankdots) [reply src/style.css:1]: tried that, did not go too well, or I did not figure out the syntax

File: src/components/NewDraftWizard/WizardComponents/WizardFooter.js (hunk 2 above)
Comment (blankdots) [reply src/components/NewDraftWizard/WizardComponents/WizardFooter.js:81]: this was just part of UI, functionality maybe later.

File: src/components/NewDraftWizard/NewDraftWizard.js [hunk 4]
```diff
@@ -61,11 +68,11 @@ const NewDraftWizard = () => {
   const nextButtonRef = useRef<Formik>(null)
 
   return (
-    <Container maxWidth={wizardStep <= 0 ? "md" : "lg"}>
-      <Paper className={classes.paper}>
+    <Container maxWidth="false" className={classes.container}>
```
Comment (otahontas) [inline src/components/NewDraftWizard/NewDraftWizard.js:71]: should be `maxWidth={false}`

**Pull request merged**